utils/test_memcheck.supp

```diff
@@ -570,3 +565,54 @@
    fun:_dl_init
    ...
 }
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Free
+   fun:free
+   ...
+   fun:dl_open_worker
+   fun:_dl_catch_error
+   fun:_dl_open
+   fun:dlopen_doit
+   fun:_dl_catch_error
+   fun:_dlerror_run
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Free
+   fun:free
+   ...
+   fun:ofi_get_core_info
+   fun:ofix_getinfo
+   fun:rxm_getinfo
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Free
+   fun:free
+   ...
+   fun:mca_base_component_repository_release
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Free
+   fun:free
+   ...
+   fun:__libc_freeres
+   fun:_vgnU_freeres
+   fun:__run_exit_handlers
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   ...
+   fun:malloc
```

Comment:
depending on why we see this one, it could also be problematic.   In general, any suppression that doesn't have any callstacks specific to test code is more of a problem because it could suppress real product issues.

Reply:
Same with this one.